pages/register/register.controller.js

```diff
@@ -0,0 +1,164 @@
+(function () {
+	angular.module('app').controller('RegisterController', ['register', 'login', 'session', '$state', '$stateParams', '$translate', RegisterController]);
+
+	function RegisterController(register, login, session, $state, $stateParams, $translate) {
+		var self = this;
+
+		// Service binding
+		self.service = register;
+
+		// Properties
+		self.username = '';
+		self.email = '';
+		self.password = '';
+		self.age = '';
+		self.terms = false;
+		self.error = '';
+		self.loading = false;
+		self.status = {
+			register: true,
+			terms: false,
+			captcha: false,
+		};
+
+		// Methods
+		self.list1 = list1;
+		self.list2 = list2;
```

Comment:
Method names `list1` and `list2` are not at all descriptives. Even watching the code, and using the app, I don't understand why are called in that way, because... actually, there are no lists anywhere.

Reply:
I was thinking about slide... or even step, both also were not enough accurate to describe the piece of form